### ISSUE TITLE
Beneath the Depths: wrong location on map

### ISSUE THREAD

**Issue by Numbermaniac** [Wynncraft/Issues#7138]
![2017-01-09_15 34 04](https://cloud.githubusercontent.com/assets/5206120/21757358/560bd02e-d681-11e6-82a7-62312ae53ccd.png)
![screen shot 2017-01-09 at 15 34 12](https://cloud.githubusercontent.com/assets/5206120/21757362/5e40dc94-d681-11e6-859f-bc1d21ae493e.png)

In the game, the book's coordinates match with the start of the quest. However, on the map it's way off.

**Comment by DebugingLonks** [comment]
The map is not 100% accurate ;P

**Comment by LordHelmchen98** [comment]
The online map only shows the surface but Javier is hidden by other parts of the island so it can't show the actual place. Also it's not directly from above so lower places hidden can be hidden by higher places that aren't above the lower place. That's why you seem to be shown at the wrong place on the map.

**Comment by Numbermaniac** [comment]
@LordHelmchen98 True, but normally the book is shown at the right location on the map, despite the parallax. In this case, it appears to be quite off.

**Comment by LordHelmchen98** [comment]
Thats because the island got revamped but not the map icons. Before the Gameplay Update the icon was correct but now after the revamp Javier is at a different position and therefore the icon if wrong. However this is by far not the only wrong icon and it's known to staff members so it will hopefully get fixed soon.
